tests/utils.go

```diff
@@ -354,6 +354,49 @@ func getPodsByLabel(label, labelType, namespace string) (*k8sv1.PodList, error)
 	return pods, nil
 }
 
+func GetProcessName(pod *k8sv1.Pod, pid string) (output string, err error) {
+	virtClient, err := kubecli.GetKubevirtClient()
+	if err != nil {
+		return
+	}
+
+	fpath := "/proc/" + pid + "/comm"
+	output, err = ExecuteCommandOnPod(
+		virtClient,
+		pod,
+		"compute",
+		[]string{"cat", fpath},
+	)
+
+	return
+}
+
+func ListCgroupThreads(pod *k8sv1.Pod) (output string, err error) {
+	virtClient, err := kubecli.GetKubevirtClient()
+	if err != nil {
+		return
+	}
+
+	output, err = ExecuteCommandOnPod(
+		virtClient,
+		pod,
+		"compute",
+		[]string{"cat", "/sys/fs/cgroup/cpuset/tasks"},
+	)
+
+	if err == nil {
+		// Cgroup V1
+		return
+	}
```

Comment:
This is a bit hacky :) We don't really know which error is being raised here.
I think it would be best to add another method to the cgroup interface that would also be used for testing.

But again, it could be made in a follow-up.

Reply:
This is how the current GetPodCPUSet in tests/utils.go works (copied the logic).
Its a little bit ugly, but pretty safe in practice i believe.

OK, can be fixed up later if desired, resolving conversation.